language/tests/regent/compile_fail/optimize_index_launch_list6.rg

```diff
@@ -12,6 +12,9 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 
+-- runs-with:
+-- [["-findex-launch-dynamic", "0"]]
```

Comment:
Is there a reason for these tests to have the dynamic check disabled?

Reply:
Yes, if we remove the flag then these tests will no longer fail (note that they are in `compile_fail`).